deps/wazuh_testing/wazuh_testing/modules/analysisd/event_monitor.py

```diff
@@ -110,3 +122,39 @@ def check_stop_dropping_events_and_credits_available_log(log_level='WARNING', ti
     """
     check_analysisd_event(callback=fr'.*{log_level}: Queues back to normal and EPS credits, no dropping events.*',
                           timeout=timeout)
+
+
+def check_if_sid_not_found(file_monitor):
+    """Check if an invalid if_sid rule value is ignored when detected
+
+    Args:
+        file_monitor (FileMonitor): Log monitor.
+    """
+
```

Comment:
Remove empty line between documentation and code

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3649/commits/80c472ee2c0de8b0c1f521d95117bedd0f8615a6